open_policy_agent/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,38 @@
+name: open-policy-agent
+files:
+- name: open_policy_agent.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/openmetrics
+  - template: instances
+    options:
+    - name: opa_url
+      required: true
+      description: Service endpoint of your Open Policy Agent instance.
+      value:
+        type: string
+        example: http://%%host%%:8181
+    - template: instances/openmetrics
+      overrides:
+        prometheus_url.value.example: http://%%host%%:8181/metrics
```

Comment:
Let's chose between `opa_url` and `prometheus_url`. You can use the `hidden` attribute if you prefer to use `opa_url`

Reply:
I am confused. Those are two different endpoints (one for the service itself, one for the metrics). In this case, they listen to the same port, but it can listen to a different one

Reply:
If it can be different.then let's keep the 2 options